src/dlls/mscoree/coreclr/README.md

```diff
@@ -0,0 +1,7 @@
+dump\_helper\_resource.bin in this folder is a text file with a single 0 byte appended on the end using a hex-editor. It is unlikely it will need to be modified frequently if at all,
+but if that changes we can always create a little nicer tooling for it. 
+
+dump\_helper\_resource.bin is used to populate the DUMP\_HELPER resource inside coreclr.dll on Windows. When an application crashes, Windows MinidumpWriteDump is planning to scan
```

Comment:
Current plan is that a future version of Windows 10 will support it, but I don't want to count my chickens before they hatch. Windows 7 and 8 support seems unlikely from what I know but it will be the OS team that gets to make a final call.